lightcycle-integration-test/src/test/java/com/soundcloud/lightcycle/sample/basic/ActivityLoggerTest.java

```diff
@@ -0,0 +1,41 @@
+package com.soundcloud.lightcycle.sample.basic;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+import org.robolectric.util.ActivityController;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = 23)
+public class ActivityLoggerTest {
+
+    @Test
+    public void verifyActivityCallback() {
+        final ActivityController<SampleActivity> controller = Robolectric.buildActivity(SampleActivity.class);
+        final SampleActivity sampleActivity = controller.get();
+        final ActivityLogger activityLogger = sampleActivity.activityLogger;
+
+        controller.create();
+        assertThat(activityLogger.isActivityCallbackCalled(ActivityLogger.ActivityLifecycleCallback.onCreate)).isTrue();
```

Comment:
The unimplemented callbacks are : onNewIntent, onOptionsItemSelected, onSaveInstanceState, onRestoreInstanceState.

And yes I think it will be better if we test them too. But I think it will be better if we separate them to other functions, so we will have test for lifecycle callback and non-lifecycle callback.

Let me know if you have any feedback :)